front-end/src/app/login/login.component.html

```diff
@@ -13,10 +13,10 @@
         </div>
         <div class="alert alert-danger" role="alert" *ngIf="incorrectLogin">Incorrect Username or Password</div>
         <div class="form-group text-center">
-          <button type="submit" class="btn btn-primary" id="login">Login</button>
+          <button class="btn btn-primary" id="login" (click)="goToProfile()">Login</button>
         </div>
         <div class="form-group text-center">
-          <button class="btn btn-primary" id="register">Register</button>
+          <button class="btn btn-primary" id="register" (click)="goToRegister()">Register</button>
```

Comment:
would be better to have one function goTo(url : string) than many functions called goToRegister, goToProfile...etc. Perhaps call (click)="this.router.router.navigate(['register']) directly in the html or create a service with the goTo function and call that.